ban/core/models.py

```diff
@@ -169,7 +175,7 @@ def compute_cia(self):
         if not get_fantoir:
             return None
         return compute_cia(str(self.parent.municipality.insee),
-                           get_fantoir(), self.number, self.ordinal)
+                           get_fantoir, self.number, self.ordinal)
```

Comment:
1. As discussed in chat, we should do another PR wich should fix get_fantoir().
2. Test_housenumber.test_get_housenumber_with_cia() was KO.
The only answer was to fix this test and push the code.
The real answer would be the 2nd PR.